rustls/Cargo.toml

```diff
@@ -16,7 +16,7 @@ build = "build.rs"
 rustversion = { version = "1.0.6", optional = true }
 
 [dependencies]
-aws-lc-rs = { version = "1.5", optional = true }
+aws-lc-rs = { version = "1.6", optional = true }
```

Comment:
1.6 is semver-compatible with 1.5 and there are no other changes in this commit that seem to require a version bump -- so why do we want this?